app/controllers/projects_controller.rb

```diff
@@ -97,7 +95,7 @@ def project_params
   end
 
   def project
-    @project
+    current_project
   end
```

Comment:
I did attempt to remove it, but his method has been registered as a helper_method, and it's being used in a lot of places (controller, erbs, helpers as well as far as I remember).

So, I ended up not going along that path to keep this PR small and to avoid a bigger risk of breaking something.